### PR TITLE
[MERGE THU 14 NOV] Add experimental option to runtime options table

### LINKED ISSUE #2256: New experimental option for Qiskit Runtime
### URL, if applicable

https://docs.quantum.ibm.com/guides/runtime-options-overview#available-options

### Describe the new content you are requesting.

We'd like to add a new experimental option to the options table. ~The final name is still TBD, and I will update the issue once that's decided~. 

The options table will have 

| Option | Sub-option | Sub-sub-option | Choices | Default | 
| -- | -- | -- | -- | -- |
| experimental |  | | | | 
|  | execution_path | | gen3-turbo* | None | 

For both Sampler and Estimator.

And the footnote would say something like "Setting `gen3-turbo` enables the next generation execution path. This is currently experimental only available on `ibm_torino`".

The in the [compatibility table](https://docs.quantum.ibm.com/guides/runtime-options-overview#options-compatibility), we should add that <new flag> cannot be used with 
- dynamic circuits
- meas_type=kerneled or meas_type=avg_kerneled
- fractional gates

### If this new content request is accepted, do you want to write the content?

I can help the team by providing enough information to write the material

### PR DESCRIPTION
**Pull request by abbycross**
Closes #2256.

~Waiting on a name for the option~ name is gen3-turbo

### CODE REVIEW

**Comment by abbycross** [comment]
All requested changes in place except for option name

File: docs/guides/runtime-options-overview.mdx [hunk 1]
```diff
@@ -135,11 +137,16 @@ Scroll to see the full table.
...
+| experimental         |                          |                         |                           |                        |                    |
+|                      | execution_path           |                         | `<new_flag>` *([see note](#experimental-option-note))*               | None                   |                    |
 
   </TabItem>
 </Tabs>
 
-
+<span id="experimental-option-note"></span>
+<Admonition type="note">
+Setting `<new flag>` enables the next-generation execution path. This is currently experimental and is only available on `ibm_torino`.
```
Comment (beckykd) [inline docs/guides/runtime-options-overview.mdx:148]: It sounds like it's experimental and temporary.  "currently" can imply that we're going to change it to be non-experimental in the future, so I'd remove that word.

Comment (abbycross) [inline docs/guides/runtime-options-overview.mdx:148]:
```suggestion
Setting `<new flag>` enables the next-generation execution path. This is experimental and is only available on `ibm_torino`.
```

File: docs/guides/runtime-options-overview.mdx [hunk 2]
```diff
@@ -135,11 +137,16 @@ Scroll to see the full table.
...
+| experimental         |                          |                         |                           |                        |                    |
+|                      | execution_path           |                         | `gen3-turbo` *([see note](#experimental-option-note))*               | None                   |                    |
 
   </TabItem>
 </Tabs>
 
-
+<span id="experimental-option-note"></span>
+<Admonition type="note">
+Setting `gen3-turbo` enables the next-generation execution path. This is experimental and is only available on `ibm_torino`.
```
Comment (jyu00) [inline docs/guides/runtime-options-overview.mdx:148]:
```suggestion
Setting `gen3-turbo` enables the next-generation execution path, which provides a great improvement in performance. This is experimental and is only available on `ibm_torino`.
```

Let's give them a little teaser on what's coming 😄. 
In the API reference, we point out experimental options are "subject to change without notification, and stability is not guaranteed". Do you think we need to mention it again here? Or would that make the sentence too long?

**Comment by beckykd** [comment]
According to @hoss Ajalloojean, this is ready to go live.

**Review by beckykd — APPROVED** [review]

**Comment by hoss** [comment]
I have not approved that this is ready to go live.

@hoss
[•]

**Comment by beckykd** [comment]
> I have not approved that this is ready to go live. @hoss [•]
Sorry, I tagged the wrong person.

**Pull request merged**